### PR TITLE
Add com.substance3d.launcher

### PR DESCRIPTION
**Pull request by FakeShemp**
https://www.substance3d.com/launcher

TODO
- [x] App renders a black screen
- [x] App crashes
- [ ] Applications cannot be installed

### CODE REVIEW

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

File: com.substance3d.launcher.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Substance Launcher
+Comment=Product aquisition for Allegorithmic Software
+Exec=substance
+Terminal=false
+Type=Application
+Icon=com.substance3d.launcher
+StartupWMClass=Substance Launcher
+Categories=GNOME;GTK;Graphics;
```
Comment (bilelmoussaoui) [inline com.substance3d.launcher.desktop]: It's not a GNOME nor a GTK app.
```suggestion
Categories=Graphics;
```

Comment (FakeShemp) [reply com.substance3d.launcher.desktop]: This is what the official .desktop file has, but yes, it makes sense to remove them.

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
The app seems stable now and Substance Source works as it should I think. However, it's not possible to install any extra apps since they want to install to `/opt`.

bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by bilelmoussaoui** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
It seems pretty final in the app that it needs to install its apps to `/opt/Allegorithmic/*`. This is not an allowed path in any way in a flatpak environment, right?

If not, I think the application needs to be patched in some way. I would appreciate a bit of help with that.

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
Looking at the logs, the app seems to download RPMs, and complain on not finding `pkexec` or `kdesudo`.

```
[6:28:19 PM] info: Download complete of https://download.substance3d.com/substance-designer/2019.x/Substance_Designer-2019.3.3-3313-linux-x64-standard.rpm
[6:28:19 PM] info: Launching product installer package
[6:28:19 PM] warn: Install canceled. Error: Error: Unable to find pkexec or kdesudo. -- Out: undefined
```

**Comment by FakeShemp** [comment]
Yes, the launcher seems to want to install apps with yum. I have no idea how to continue.

**Comment by FakeShemp** [comment]
With this update I get to 
```
[11:24:02 PM] info: Launching product installer package
[11:24:02 PM] warn: Install canceled. Error: Error: User did not grant permission. -- Out: 
```
which is actually as far as I get running the AppImage as well.

**Comment by nedrichards** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Comment by FakeShemp** [comment]
bot, build com.substance3d.launcher

**Pull request closed without merging**